### ISSUE TITLE
Tartube won't start in Windows 10 if Python 2 already installed

### ISSUE THREAD

**Issue by DogSledder** [axcore/tartube#271]
Thanks for taking the time to submit a bug report! Please include the following information:

### What operating system are you using?
Windows 10 Pro 20H2
### What version of Tartube are you using?
v2.3.110

### What happens when you run Tartube from a terminal window?
```
$ python3 tartube/tartube
Python path configuration:
  PYTHONHOME = 'C:\Python27'
  PYTHONPATH = (not set)
  program name = 'C:\Users\jpj\AppData\Local\Tartube\msys64\mingw64\bin\python3.exe'
  isolated = 0
  environment = 1
  user site = 1
  import site = 1
  sys._base_executable = 'C:/Users/jpj/AppData/Local/Tartube/msys64/mingw64/bin/python3.exe'
  sys.base_prefix = 'C:\\Python27'
  sys.base_exec_prefix = 'C:\\Python27'
  sys.executable = 'C:/Users/jpj/AppData/Local/Tartube/msys64/mingw64/bin/python3.exe'
  sys.prefix = 'C:\\Python27'
  sys.exec_prefix = 'C:\\Python27'
  sys.path = [
    'C:/Python27/lib/python38.zip',
    'C:\\Python27/lib/python3.8',
    'C:\\Python27/lib/python3.8',
    'C:\\Python27/lib/python3.8/lib-dynload',
  ]
Fatal Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding
Python runtime state: core initialized
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00003d40 (most recent call first):
<no Python frame>
```
I fixed this by editing "tartube_64bit.bat" to remove the PYTHONHOME environment variable from the local environment:
```
set PYTHONHOME=
..\..\..\usr\bin\mintty.exe -w hide /bin/env MSYSTEM=MINGW64 /bin/bash -lc /home/user/tartube/tartube_mswin.sh

```
After this change Tartube started and I was able to continue setup.

This appears to be the same as the bug reported in Issue #79 .

I don't know if this is limited to just a previous installation of Python 2 or if another version of Python would cause the same bug to appear.

**Comment by axcore** [comment]
I tested this by intsalling Python2 on a clean Win10 box, then installing Tartube on top of that. Tartube started normally without the issue you describe. Anyway, I've added your change to both .bat files which should prevent the same problem in the future.

**Comment by DogSledder** [comment]
I vaguely remember installing Python 2 as part of another package's setup instructions. I think that it called for setting environment variables or a script did it. I just followed the recipe. It was quite a while ago. I assumed that's how Python 2 is set up.

I just did a quick web search for PYTHONHOME issues and found warnings about using it because of cases like this. I also found that there are command line options for Python (-E or -I) that tell it to ignore environment variables. Maybe one of those would be a better fix? The -I option (Run Python in isolated mode) sounds like it addresses this class of issues.

Anyhow, thanks for this handy program.
